modules/generic/proto_fields.lua

```diff
@@ -33,15 +33,15 @@ local proto_pfields = {
     tag      = ProtoField.uint32("protobuf.tag", "Tag", base.DEC, nil, nil, "Field tag value"),
     wiretype = ProtoField.uint8 ("protobuf.wiretype", "Wire Type", base.DEC, wiretype_valstring, 0x07, "Field wire type value"),
     length   = ProtoField.uint32("protobuf.length", "Length", base.DEC, nil, nil, "Field length"),
-    value    = ProtoField.bytes ("protobuf.value", "Value", base.HEX, nil, nil, "Field value bytes"),
+    value    = ProtoField.bytes ("protobuf.value", "Value", base.NONE, nil, nil, "Field value bytes"),
```

Comment:
Why? Do you expect this field to always be NULL?

Reply:
I will fix this later.
I get an error so bytes object can't get `base.HEX`